Mailerlite

maxharrisnet opened the pull request:
- Adding the Mailerlite global code snippet
- Swapping the form in the footer

maxharrisnet commented on 404.html:
```diff
@@ -1,266 +1,252 @@
 <!DOCTYPE html>
```
@SolivanLau hide whitespace

maxharrisnet commented on 404.html:
```diff
@@ -1,266 +1,252 @@
+				</nav>
+
+				<!-- CAT Newsletter -->
+				<div class="ml-embedded" data-form="w80VC7"></div>
```
Replacing the form with Mailerlite target div

maxharrisnet commented on 404.html:
```diff
@@ -1,266 +1,252 @@
+	<!-- styles -->
+	<link rel="stylesheet" href="css/global.css" />
+	<link rel="stylesheet" href="css/404.css" />
+	<!-- MailerLite Universal -->
```
Adding MailerLite script to all pages

maxharrisnet commented on css/global.css:
```diff
@@ -888,23 +888,96 @@ footer .social {
+	fill: var(--color-white);
+}
+
+.ml-embedded,
```
It's so ugly, don't look @SolivanLau !

SolivanLau commented on 404.html:
```diff
@@ -1,266 +1,252 @@
+	<link rel="stylesheet" href="css/global.css" />
+	<link rel="stylesheet" href="css/404.css" />
+	<!-- MailerLite Universal -->
+	<script>
```
looks good - can you please add this to **main.js** or another **component.js** file? I think it is easier to maintain this way.

maxharrisnet commented:
@SolivanLau  I just made a few changes:
- Replaced code block with a script file
- Added form to policy pages (I think I forgot to merge them in before doing a find/replace)
- Styling the checkmark to be large
